### PR TITLE
Update code documentation and resolve a module incompatibility

### PR DESCRIPTION
**Pull request by Dhananjhay**
This PR primarily updates the code documentation and README file to organize the repository's development. It also resolves simpleITK module incompatibility with the rest of the package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,84 @@
-# autoafids_prep
- snakemake workflow to prep data for autoafids project
+# Anatomical Fiducials (AFIDs) Data Processing Pipeline 
+Robarts Research Institute Team - 2023-2024
```
Comment (ataha24) [inline README.md]: update to: 
"AIMS Lab Research Team at the Robarts Research Institute - 2023-2024"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,84 @@
...
+
+
+*This package is under active development. It should be stable and reproducible, but please let any of the active contributing members know if there are any bugs or unusual behaviour.*
+
+This Python package is a data processing pipeline based on Snakemake and SnakeBIDS workflow management tools to prepare data for a Convolutional Neural Network (CNN) deep learning model [AFIDS-CNN](https://github.com/afids/afids-CNN). Since AFIDS-CNN is under active development, this package contains tunable parameters that are not normally exposed in a data processing pipeline; the user is highly encourage to read docstrings and get familiar with the relevant workflow managements tools prior to using this software. Likewise, there may be frequent updates to this package as the project matures (see the [changelog](CHANGELOG.md) for more details).
```
Comment (ataha24) [inline README.md]: AFIDS-CNN --> afids-CNN

File: README.md [hunk 3]
```diff
@@ -1,2 +1,84 @@
...
+
+## Brief Overview of the Pipeline
+1. Performs rigid registraion to MNI template.
```
Comment (ataha24) [inline README.md]: 1. Performs rigid registration (i.e., 6 d.o.f) from MNI template to Native space.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,84 @@
...
+2. Conformes images to 1 mm isotropic resolution.
```
Comment (ataha24) [inline README.md]: 2. Conformes images to a desired isotropic resolution (default: 1mm).

File: README.md [hunk 5]
```diff
@@ -1,2 +1,84 @@
...
+
+
+*This package is under active development. It should be stable and reproducible, but please let any of the active contributing members know if there are any bugs or unusual behaviour.*
+
+This Python package is a data processing pipeline based on Snakemake and SnakeBIDS workflow management tools to prepare data for a Convolutional Neural Network (CNN) deep learning model [AFIDS-CNN](https://github.com/afids/afids-CNN). Since AFIDS-CNN is under active development, this package contains tunable parameters that are not normally exposed in a data processing pipeline; the user is highly encourage to read docstrings and get familiar with the relevant workflow managements tools prior to using this software. Likewise, there may be frequent updates to this package as the project matures (see the [changelog](CHANGELOG.md) for more details).
+
+## Brief Overview of the Pipeline
+1. Performs rigid registraion to MNI template.
+2. Conformes images to 1 mm isotropic resolution. 
+3. Executes intensity normalization (i.e., White Matter (WM) to 110) followed by min-max normalization.
```
Comment (ataha24) [inline README.md]: 3. Executes image intensity normalization by various supported algorithms (default: min-max normalization).

File: autoafids_prep/workflow/scripts/tform_script.py [hunk 6]
```diff
@@ -8,7 +8,19 @@
...
+    """
+    Determines if the fcsv file uses Right-Anterior-Superior (RAS) or Left-Posterior-Superior (LPS) coordiante system.
+
+    Parameters
+    ----------
+        input_fscv:: str
+            filename
+    Returns
+    -------
+        coord_sys:: ??? (Ask Alaa)
```
Comment (ataha24) [inline autoafids_prep/workflow/scripts/tform_script.py]: str type

**Review by ataha24 — CHANGES_REQUESTED** [review]
I tested the workflow on CBS and performed as expected. I left some responses and comments but all very minor. We should be ready to merge this.

**Comment by Dhananjhay** [comment]
All the above mentioned changes have been made. Ready to merge whenever you are!

**Review by ataha24 — APPROVED** [review]
Looks good!

**Pull request merged**